core/src/main/java/hudson/XmlFile.java

```diff
@@ -126,8 +129,18 @@ public XmlFile(File file) {
     }
 
     public XmlFile(XStream xs, File file) {
+        this(xs, file, true);
+    }
+
+    /**
+     * @param force Whether or not to flush the page cache to the storage device with {@link
+     *     FileChannel#force} (i.e., {@code fsync}} or {@code FlushFileBuffers}) before this method
+     *     returns. If you set this to {@code false}, you will lose data integrity.
```

Comment:
this comment is inocreect. 
neither ` FileChannel#force` nor `FlushFileBuffers` will be called before this method returns.
1) this is a constructor not a method
2) there is no data written here.